tests/common/include/tests/common/tools/create_toy_geometry.hpp

```diff
@@ -412,6 +445,15 @@ void create_endcap_modules(context_t &ctx, const dindex volume_id,
                                                   m_local_x);
         }
     }
+
+    // add surface grid
+    // TODO: WHat is the proper value of n_phi_bins?
```

Comment:
We should probably discuss this with Andi in the next meeting, also because he did the matching of the phi bins in the ODD endcaps by hand, if I'm not mistaken

Reply:
we can just leave the values for the moment, and change it when needed

Reply:
Maybe we should also consider this at some later point:  detray/core/include/detray/tools/bin_association.hpp ? I don't know if it is helpful or not